smk/binning_preparation.smk

```diff
@@ -280,6 +280,16 @@ def get_assemblies_for_scaf_type(wildcards):
 
     return(asms)
 
+def calculate_batch_filesize(wildcards):
+    filtered_batch_size = (max_job_ram_gb-2)/22*1024
+    if wildcards.scaf_type.endswith("nanopore"):
+        # assuming that size filtering doesn't drastically reduce file size
+        return(round(filtered_batch_size))
+    else:
+        # 2500bp filtering reduces file size by approx. half,
+        # we account for lower min. length and better assemblies here
+        return(round((filtered_batch_size-50)/0.6))
```

Comment:
Could we please catch negative values for `size - 50`? If someone sets max-mem as `3GB`, it would end up with negative values. Or we can catch it at the conf-parsing level and reject values below `4GB`.